.github/sync-repo-settings.yaml

```
@@ -0,0 +1,37 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Synchronize repository settings from a centralized config
+# https://github.com/googleapis/repo-automation-bots/tree/main/packages/sync-repo-settings
+# Install: https://github.com/apps/sync-repo-settings
+
+# Disable merge commits
+rebaseMergeAllowed: true
+squashMergeAllowed: true
+mergeCommitAllowed: false
+# Enable branch protection
+branchProtectionRules:
+- pattern: main
+  isAdminEnforced: true
+  requiredStatusCheckContexts:
```

I missed this first time around, there's a missing line here
```suggestion
  requiredStatusCheckContexts:
      - 'cla/google'
```

---

Soo... actually I think I took that out because it seems to be already enforced at the org level? Or at least it's been already running on each PR so far.

---

These sorts of templates should be copied verbatim to ensure compliance.